collection-spec/collection-spec.md

```diff
@@ -172,17 +172,17 @@ This is done where there is not a clear official option, or where STAC uses an o
 
 | Type    | Description                                                  |
 | ------- | ------------------------------------------------------------ |
-| self    | STRONGLY RECOMMENDED. *Absolute* URL to the location that the collection file can be found online, if available. This is particularly useful when in a download package that includes metadata, so that the downstream user can know where the data has come from. |
+| self    | STRONGLY RECOMMENDED. *Absolute* URL to the location that the Collection file can be found online, if available. This is particularly useful when in a download package that includes metadata, so that the downstream user can know where the data has come from. |
 | root    | URL to the root STAC Catalog or Collection. Collections should include a link to their root, even if it's the root and points to itself. |
 | parent  | URL to the parent STAC Catalog or Collection. Non-root Collections should include a link to their parent. |
 | child   | URL to a child STAC Catalog or Collection. |
-| item    | URL to a STAC Item. All Items linked from a Collection MUST refer back to its Collection with the [`collection` relation type](../item-spec/item-spec.md#relation-types). |
+| item    | URL to a STAC Item. All Items linked from a Collection MUST refer back to its Collection with the [`Collection` relation type](../item-spec/item-spec.md#relation-types). |
```

Comment:
That must be lower-cased!

Reply:
ah woops I thought I fixed all of them -- I had some conflicts to resolve from the capitalization PR but missed some when I went back to fix

Reply:
e77843a